demo/README.md

```diff
@@ -23,7 +23,8 @@
 	       - `brew install kind` on macOS
 	    - Provision a local cluster and registry in Docker: `make kind-up`
 	- **Option 2:** A Kubernetes cluster - use an already provisioned cluster config, either in the default location ($HOME/.kube/config) or referenced by the $KUBECONFIG environment variable.
-
+      - Set `CTR_REGISTRY` in your `.env` file to a container registry you have permission to push and pull from. 
```

Comment:
If your registry is not Dockerhub or localhost, you also need to `docker login <registry url>` first before running the demo script. I remember when I tried with Azure CR, this happened to me

Reply:
Good note! I have added a bullet to address this